testsuite/src/test/java/org/sonatype/nexus/maven/staging/test/support/StagingMavenPluginITSupport.java

```diff
@@ -150,6 +154,11 @@ private Properties getDefaultProperties(final String repository,
   protected void initialiseVerifier(final File projectRoot) throws Exception {
     String settingsXml = new File(projectRoot, "preset-nexus-maven-settings.xml").getAbsolutePath();
 
+    Path m2Home = getM2Home();
```

Comment:
This code attempts to get the M2 home from where the wrapper installs it - I just quickly threw it together so it probably needs some renaming and constants etc.

Reply:
🏅